aep/general/oas.yaml

```diff
@@ -0,0 +1,501 @@
+    get:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+        - in: query
+          name: max_page_size
+          required: true
+          schema: {}
+          type: integer
+        - in: query
+          name: page_token
+          required: true
+          schema: {}
+          type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                properties:
+                  results:
+                    items:
+                      $ref: '#/components/schemas/book'
+                    type: array
+                  unreachable:
+                    items:
+                      type: string
+                    type: array
+                type: object
+          description: Successful response
+    post:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+        - in: query
+          name: id
+          required: true
+          schema: {}
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/book'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/book'
+          description: Successful response
+  /publishers/{publisher}/books/{book}:
+    delete:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+        - in: path
+          name: book
+          required: true
+          schema: {}
+          type: string
+      responses:
+        '200':
+          content: null
+          description: ''
+    get:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+        - in: path
+          name: book
+          required: true
+          schema: {}
+          type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/book'
+          description: Successful response
+    patch:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+        - in: path
+          name: book
+          required: true
+          schema: {}
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/book'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/book'
+          description: Successful response
+    put:
```

Comment:
Do we have `Apply` yet?  If not, should we have `PUT` methods?

Reply:
this example isn't really visible *except* for the other snippets, but I agree let's not add it until it's official